### PR TITLE
Resolutions, hashes and .gitignore and fixed language problems

### PR DESCRIPTION
**Pull request by tkovs**

### CODE REVIEW

File: Problem039/C++/Makefile [hunk 1]
```diff
@@ -1,7 +1,7 @@
-CXX = g++
```
Comment (rafaelcn) [inline Problem039/C++/Makefile]: If it is C++ why did you change it for gcc?

File: Problem039/C/Makefile [hunk 2]
```diff
@@ -1,7 +1,7 @@
-CXX = g++
+CXX = gcc
 LFLAGS = -lm -Wall -o
 TARGET =  solution_1
-EXTENSION_TARGET = cpp
```
Comment (rafaelcn) [inline Problem039/C/Makefile]: Same applies here.

File: Problem039/C++/Makefile (hunk 1 above)
Comment (tkovs) [inline Problem039/C++/Makefile]: This code is written in C.

Comment (tkovs) [inline Problem039/C++/Makefile]: [It](https://github.com/DestructHub/ProjectEuler/blob/master/Problem039/C%2B%2B/solution_1.cpp) is pure c.

Comment (rafaelcn) [inline Problem039/C++/Makefile]: Why does it remains in the C++ folder then?

Comment (tkovs) [inline Problem039/C++/Makefile]: I [renamed](https://github.com/DestructHub/ProjectEuler/pull/69/commits/7f3da85f1a38734acf151ab4687657e8d461fd27) the directory. It was an someone's error.

Comment (rafaelcn) [inline Problem039/C++/Makefile]: Thanks @tkovs! I really appreciate.

**Comment by rafaelcn** [comment]
Approved.

**Comment by ryukinix** [comment]
Approved?
